<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>683</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>ToDo List</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>appIcon.png</normaloff>appIcon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(103, 61, 255, 255), stop:0.505682 rgba(0, 147, 246, 255), stop:1 rgba(49, 58, 255, 255));
font-family: Ruda Regular;</string>
  </property>
  <widget class="QWidget" name="mainFrame">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="leftMargin">
     <number>15</number>
    </property>
    <property name="topMargin">
     <number>15</number>
    </property>
    <property name="rightMargin">
     <number>15</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <item>
     <widget class="QLabel" name="nameLable">
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255,255,255,30);
border: 1px solid rgba(255,255,255,40);
border-radius: 7px;
color: white;
font-size: 25px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="text">
       <string>Список дел</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListView" name="reminderList">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255,255,255,30);
border: 1px solid rgba(255,255,255,40);
border-radius: 7px;
color: white;
font-size: 25px;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="buttonsFrame">
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255,255,255,0)</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QPushButton" name="changeButton">
         <property name="styleSheet">
          <string notr="true">QPushButton{
color: white;
background-color: rgba(255,255,255,30);
border: 1px solid rgba(255,255,255,40);
border-radius: 7px;
width: 120px;
height: 50px;
}

QPushButton:hover{
background-color: rgba(255,255,255,40)
}

QPushButton:pressed{
background-color: rgba(255,255,255,70)
}</string>
         </property>
         <property name="text">
          <string>Изменить</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/icons/icons/edit_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/icons/icons/edit_FILL0_wght400_GRAD0_opsz24.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="completeButton">
         <property name="styleSheet">
          <string notr="true">QPushButton{
color: white;
background-color: rgba(255,255,255,30);
border: 1px solid rgba(255,255,255,40);
border-radius: 7px;
width: 100px;
height: 50px;
}

QPushButton:hover{
background-color: rgba(255,255,255,40)
}

QPushButton:pressed{
background-color: rgba(255,255,255,70)
}</string>
         </property>
         <property name="text">
          <string>Выполнено</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addButton">
         <property name="styleSheet">
          <string notr="true">QPushButton{
color: white;
background-color: rgba(255,255,255,30);
border: 1px solid rgba(255,255,255,40);
border-radius: 7px;
width: 100px;
height: 50px;
}

QPushButton:hover{
background-color: rgba(255,255,255,40)
}

QPushButton:pressed{
background-color: rgba(255,255,255,70)
}</string>
         </property>
         <property name="text">
          <string>Добавить</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/icons/icons/add_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/icons/icons/add_FILL0_wght400_GRAD0_opsz24.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteButton">
         <property name="styleSheet">
          <string notr="true">QPushButton{
color: white;
background-color: rgba(255,255,255,30);
border: 1px solid rgba(255,255,255,40);
border-radius: 7px;
width: 120px;
height: 50px;
}

QPushButton:hover{
background-color: rgba(255,255,255,40)
}

QPushButton:pressed{
background-color: rgba(255,255,255,70)
}</string>
         </property>
         <property name="text">
          <string>Удалить</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/icons/icons/delete_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/icons/icons/delete_FILL0_wght400_GRAD0_opsz24.svg</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="shareButton">
      <property name="styleSheet">
       <string notr="true">QPushButton{
color: white;
background-color: rgba(255,255,255,30);
border: 1px solid rgba(255,255,255,40);
border-radius: 7px;
width: 30px;
height: 30px;
}

QPushButton:hover{
background-color: rgba(255,255,255,40)
}

QPushButton:pressed{
background-color: rgba(255,255,255,70)
}</string>
      </property>
      <property name="text">
       <string>LiveShare</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/icons/icons/share_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/icons/icons/share_FILL0_wght400_GRAD0_opsz24.svg</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
